network-api/networkapi/buyersguide/factory.py

```diff
@@ -319,6 +318,21 @@ def generate(seed):
             related_product.save()
             product_page.related_product_pages.add(related_product)
 
+            # Create new ProductUpdates orderable for each PNI product
+            product_update = ProductUpdates(
+                page=product_page,
+                update=ProductUpdateFactory()
+            )
+            product_update.save()
+            product_page.updates.add(product_update)
+
+            # Create three new privacy policy links for each PNI product
+            privacy_orderable = ProductPagePrivacyPolicyLinkFactory(
+                page=product_page,
+            )
+            privacy_orderable.save()
+            product_page.privacy_policy_links.add(privacy_orderable)
```

Comment:
Privacy link orderables. I snuck this in here since we needed it and I was in here anyway.

Reply:
so sneaksies